sdk/README.md

```diff
@@ -0,0 +1,22 @@
+# Contents of `sdk/`
+
+**All the DSP related files.**
+
+This folder contains the C code which is ran on the DSP on the AMDC hardware.
+
+Each folder is a complete project which can be compiled and ran on the Xilinx Zynq-7000 SoC. There are currently two projects:
+
+1. `basic` &mdash; C code written by Eric Severson for his PhD dissertation (*archived and not maintained*)
+
+- Designed for BM control on PECB platform (similar to AMDC, but based on ZedBoard; developed at UMN)
+- Control performed in ISR context
+- Real-time Ethernet (lwIP) / serial logging capabilities
+
+2. `bare` &mdash; C code written by Nathan Petersen at UW-Madison (**actively maintained**)
```

Comment:
> C code written by Nathan Petersen at UW-Madison (**actively maintained**)

Great to delineate that this is the actively maintained code. It is okay to mention your name here as the author -- but I want to raise the points that 1) authorship is indicated in the commit record; and 2) our goal is that eventually you aren't the sole author of this. However, I like the idea of identifying you as the *maintainer*. What do you think?

I have been mulling over GitHub's ability to denote [code owners](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners). GitHub allows us to set up repo settings so that merges into `develop` / `master` can only happen with an approving review of the code owners (right now we have it set up so that a developer just needs `x` reviewers from the Severson group). It seems like identifying a maintainer for our different repos (or sections of repos) would be advantageous for reviewing new changes and helping to set a vision for where the projects are going. 

Thoughts?

Reply:
@elsevers Good points. I like the idea of having maintainers for different projects and having that enforced in GitHub. We should make a list in the KB of projects and who their maintainer is (point of contact).

I'm happy to remove my name from the doc, doesn't bother me. I was just trying to give people indication as to who to talk to about the code if they had questions. But eventually, they won't be able to talk to me! So yes, I'll remove my name.

Reply:
Resolved in latest commits.